r/vignettes/arrow.Rmd

```diff
@@ -120,8 +120,8 @@ In the tables, entries with a `-` are not currently implemented.
 | float32           | double                    |
 | float64           | double                    |
 | utf8              | character                 |
-| binary            | vctrs::vctrs_list_of(raw) |
-| fixed_size_binary | -                         |
+| binary            | arrow_binary              |
```

Comment:
Should footnote and explain what these `arrow_` types are (at least that they're subclasses of list/vctrs::list_of)

Reply:
That's no longer the case for the `*binary` ones, they inherit from `vctrs_vctr` mostly for printing purposes. I'll add some footnotes.